service/controller/v3/resource/deployment/resource.go

```diff
@@ -5,12 +5,13 @@ import (
 	"fmt"
 
 	azureresource "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-02-01/resources"
+	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 
 	"github.com/giantswarm/azure-operator/client"
 	"github.com/giantswarm/azure-operator/service/controller/setting"
-	"github.com/giantswarm/azure-operator/service/controller/v3/cloudconfig"
+	"github.com/giantswarm/azure-operator/service/controller/v3/controllercontext"
```

Comment:
The deployment does not need the `cloudconfig` service anymore because the VMSSs are managed in the `instance` resource. Here we only need the controller context because we need to dispatch some information to the `instance` resource where the VMSSs are managed.